pkg/action/cobra.go

```diff
@@ -7,13 +7,14 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/launchrctl/launchr/internal/launchr"
 	"github.com/launchrctl/launchr/pkg/cli"
 	"github.com/launchrctl/launchr/pkg/jsonschema"
 	"github.com/launchrctl/launchr/pkg/log"
 )
 
 // CobraImpl returns cobra command implementation for an action command.
-func CobraImpl(a *Action, streams cli.Streams) (*cobra.Command, error) {
+func CobraImpl(a *Action, streams cli.Streams, processors map[string]launchr.ValueProcessor) (*cobra.Command, error) {
```

Comment:
If you define processors on `action.Manager`, you won't need to process values on Cobra. Everything will be handled by the manager.